components/Layout/Layout.styled.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+export const NavBar = styled.nav`
+  position: fixed;
+  bottom: 0.2rem;
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  padding: 0 1rem;
+  background-color: transparent;
```

Comment:
You could think about a background-color as you navigation icons are harder to identify when other text is directly below the icons